social_django/fields.py

```diff
@@ -23,8 +26,13 @@
 
 field_class = functools.partial(six.with_metaclass, field_metaclass)
 
+if getattr(settings, setting_name('POSTGRES_JSONFIELD'), False):
```

Comment:
Is this setting ever documented?  It really seems like a personal hack to me...

Reply:
I think this need to be documented, but I don't feel like it's a hack

it's a useful extension ONLY for postgres db

Reply:
I thin it was my fault not to add it do documentation, maybe because I didnt have proper information on documentation guidelines for pull request

Reply:
My point being: relying on a boolean value `SOCIAL_AUTH_POSTGRES_JSONFIELD` may not be the best approach, even if it was documented.

Say we could probably have something like `SOCIAL_AUTH_JSONFIELD_CLASS`, default to `'django.db.models.TextField'`.  It could then be redefined to `'django.contrib.postgres.fields.JSONField'` by user.  In my opinion this is better than the boolean value, as it is more explicit.

Reply:
Yes, I can agree that probably direct class definition is actually better